conda_config.py

```diff
@@ -0,0 +1,38 @@
+import yaml, argparse, os
+
+
+class CondaDumper(yaml.Dumper):
+
+    def increase_indent(self, flow=False, indentless=False):
+        return super(CondaDumper, self).increase_indent(flow, False)
```

Comment:
Why is this necessary? Is this just a style thing, or is it a functional thing?

Reply:
Unfortunately it is functional.  The existing recipe is actually a Jinja2 template masquerading as a yaml file.  Whatever they are using to parse that template requires that we have the increased indents even though the YAML spec does not.